packages/curve-ui-kit/src/hooks/useFeatureFlags.ts

```diff
@@ -25,4 +25,4 @@ export const useDexMarketList = useBetaChannel
 export const useBorrowUnifiedForm = useBetaChannel
 
 /** New user profile button on the header */
-export const useUserProfileButton = () => useStableChannel() && !isCypress
+export const useUserProfileButton = useStableChannel
```

Comment:
tbh i think this could be removed? it's been stable for a while now, it was just gated from e2e tests